server/config/config.go

```diff
@@ -55,6 +55,7 @@ type ServerConfig struct {
 	TLS        bool
 	TLSProfile string // TODO #271 set `yaml:"tls_compatibility"`
 	URLPrefix  string `yaml:"url_prefix"`
+	KeepAlive  bool   `yaml:"keepalive"`
```

Comment:
Nit: Can the struct member name be `Keepalive` for consistency with the camel-case/snake-case transformation others use?

Reply:
ohh yea, no prob.